examples/security/graphdb-users.yaml

```diff
@@ -0,0 +1,27 @@
+# This YAML file defines a Kubernetes Secret named "graphdb-users."
+# It stores the contents of the "users.js" file (located in the "files" directory)
+# in base64-encoded format. This Secret securely provides sensitive data,
+# such as configuration files, to Kubernetes pods.
+#
+# Place this Secret under the templates directory of the Helm chart.
```

Comment:
The common use case is to deploy the chart from our repository, not by cloning this repository. 
Change the comment to create this beforehand with `kubectl apply -f ..`